### PR TITLE
CMS: 2 new tools records

### PR DESCRIPTION
**Pull request by jmhogan**
Adding records for the NanoAODRun1Examples (12506) and nanoAOD-tools (12507) code repositories. 
Corresponding link changes in the NanoAOD getting started page. 

The zip files are in /eos/opendata/cms/upload/: 
NanoAODRun1Examples-1.0.zip
nanoAOD-tools-1.0.zip

### CODE REVIEW

File: cernopendata/modules/fixtures/data/docs/cms-getting-started-nanoaod/cms-getting-started-nanoaod.md [hunk 1]
```diff
@@ -149,18 +149,18 @@ Before exiting the ROOT session, your histogram canvas should look something lik
 
 #### NanoAOD skimming and analysis with `NanoAOD-tools`
 
-The [NanoAOD-tools](https://github.com/cms-nanoAOD/nanoAOD-tools/) repository provides a toolkit to perform operations on NanoAOD files outside the CMSSW software environment. This program can be used to:
+The [NanoAOD-tools](/record/12507) repository provides a toolkit to perform operations on NanoAOD files outside the CMSSW software environment. This program can be used to:
```
Comment (katilp) [inline cernopendata/modules/fixtures/data/docs/cms-getting-started-nanoaod/cms-getting-started-nanoaod.md:153]: If we really go into details, the record is not a repository now so maybe we need to change to `The [NanoAOD-tools](https://github.com/cms-nanoAOD/nanoAOD-tools/) record`?

File: cernopendata/modules/fixtures/data/docs/cms-getting-started-nanoaod/cms-getting-started-nanoaod.md [hunk 2]
```diff
@@ -149,18 +149,18 @@ Before exiting the ROOT session, your histogram canvas should look something lik
...
 
 * skim events that fail the validation
 * skim events that fail an analysis condition based on NanoAOD branch contents
 * drop branches from the NanoAOD file to reduce its size
 * compute many common corrections for simulation
 * compute new observables in user-created modules
 
-Skimming data events that fail the validation is a critical feature, so let's see how to do it by following instructions from the [NanoAOD-tools](https://github.com/cms-nanoAOD/nanoAOD-tools/) page. This example will be performed in the ROOT docker container using files from the [Tau 2016H dataset](/record/30565).
+Skimming data events that fail the validation is a critical feature, so let's see how to do it by following instructions from the [NanoAOD-tools](/record/12507) repository README. This example will be performed in the [ROOT docker container](/docs/cms-guide-docker#nanoaod) using files from the [Tau 2016H dataset](/record/30565). The instructions here download files via `git clone`, but it is also possible to download a `.zip` file of the repository from the [record page](/record/12507)
```
Comment (katilp) [inline cernopendata/modules/fixtures/data/docs/cms-getting-started-nanoaod/cms-getting-started-nanoaod.md:160]: Here we need to change the wording as the record does not have README (or link to the repository)

**Review by katilp — CHANGES_REQUESTED** [review]
Thanks, some comments added.

File: cernopendata/modules/fixtures/data/docs/cms-getting-started-nanoaod/cms-getting-started-nanoaod.md (hunk 2 above)
Comment (jmhogan) [reply cernopendata/modules/fixtures/data/docs/cms-getting-started-nanoaod/cms-getting-started-nanoaod.md:160]: Now linking like "NanoAOD-tools [repository](link to github)" and then at the end of the paragraph I've said "zip file of the NanoAOD-tools repository from the [record page]()"

File: cernopendata/modules/fixtures/data/records/cms-tools-nanoaodtools-skimmer.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      "Run2016G",
+      "Run2016H",
+    ],
+    "source_code_repository": {
+      "url": "https://github.com/cms-opendata-analyses/nanoAOD-tools"
+    },
+    "system_details": {
+      "release": "This software requires python 2.7 and a recent ROOT version."
+    },
+    "title": "",
```
[the remark this answers is not in the document]
Comment (jmhogan) [reply cernopendata/modules/fixtures/data/records/cms-tools-nanoaodtools-skimmer.json:37]: Whoops! Forgot a title 😆 Using this suggestion

File: cernopendata/modules/fixtures/data/docs/cms-getting-started-nanoaod/cms-getting-started-nanoaod.md (hunk 1 above)
Comment (jmhogan) [reply cernopendata/modules/fixtures/data/docs/cms-getting-started-nanoaod/cms-getting-started-nanoaod.md:153]: Changing this one to "the nanoaod-tools software" (where the software can be found on that record)

File: cernopendata/modules/fixtures/data/records/cms-tools-Run1-nano-examples.json [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    },
+    "system_details": {
+      "release": "ROOT 6.30 or later"
+    },
+    "title": "Analysis examples using CMS 2010, 2011, or 2012 derived data in the NanoAODRun1 format",
+    "type": {
+      "primary": "Software",
+      "secondary": [
+        "Analysis",
+        "Workflow"
```
[the remark this answers is not in the document]
Comment (jmhogan) [reply cernopendata/modules/fixtures/data/records/cms-tools-Run1-nano-examples.json]: Done (and in the other tool)

File: cernopendata/modules/fixtures/data/records/cms-tools-nanoaodtools-skimmer.json [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+      "secondary": [
+        "Analysis",
+        "Workflow"
+      ]
+    },
+    "usage": {
+      "description": "<p>This software can be run with a plain ROOT installation. Go to <a href=\"https://root.cern\">root.cern</a> for instructions how to install the software, or use the ROOT docker container documented on the <a href=\"/docs/cms-guide-docker\">CMS docker guide</a>.</p><p>To download the files, you can either use directly the web browser or follow the instructions given in the <a href=\"/docs/cms-getting-started-nanoaod\">NanoAOD Getting Started page</a>.The Getting Started page provides an example for using NanoAOD-tools to skim NanoAOD and keep only the validated runs, and other examples are provided in the README file of the repository.</p>"
+    },
+    "use_with": {
+      "description": "This software can be run with the datasets in the nanoaod or nanoaodsim formats",
```
[the remark this answers is not in the document]
Comment (jmhogan) [reply cernopendata/modules/fixtures/data/records/cms-tools-nanoaodtools-skimmer.json]: Done

File: cernopendata/modules/fixtures/data/records/cms-tools-nanoaodtools-skimmer.json [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    "authors": [
+      {
+        "name": "CMS Open data group"
+      }
+    ],
+    "collections": [
+      "CMS-Tools"
+    ],
+    "date_created": [
+      "2016",
```
[the remark this answers is not in the document]
Comment (jmhogan) [reply cernopendata/modules/fixtures/data/records/cms-tools-nanoaodtools-skimmer.json]: Done

File: cernopendata/modules/fixtures/data/records/cms-tools-nanoaodtools-skimmer.json [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+      "CMS"
+    ],
+    "license": {
+      "attribution": "GNU General Public License (GPL) version 3"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "12507",
+    "run_period": [
+      "Run2016G",
+      "Run2016H",
```
[the remark this answers is not in the document]
Comment (jmhogan) [reply cernopendata/modules/fixtures/data/records/cms-tools-nanoaodtools-skimmer.json]: Done

File: cernopendata/modules/fixtures/data/records/cms-tools-Run1-nano-examples.json [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+      ]
+    },
+    "usage": {
+      "description": "<p>The analysis can be run with a plain ROOT installation. Go to <a href=\"https://root.cern\">root.cern</a> for instructions how to install the software, or use the ROOT docker container documented on the <a href=\"/docs/cms-guide-docker\">CMS docker guide</a>.</p><p>To download the files, you can either use directly the web browser or the following command.</p><p><code>git clone git://github.com/cms-opendata-analyses/NanoAODRun1Examples</code></p>><p>To run an example, enter one of the <code>dimuon_20XX</code> directories and execute the ROOT macro inside with a command such as:</p><p><code>$ root -l Dimuon2011_eospublic.C</code></p><p>Step-by-step commands and output examples are provided on the <a href=\"https://github.com/cms-opendata-analyses/NanoAODRun1Examples\">Github repository</a>. The final plots of the analysis are also included in the source code repository.</p>"
```
[the remark this answers is not in the document]
Comment (jmhogan) [reply cernopendata/modules/fixtures/data/records/cms-tools-Run1-nano-examples.json]: Done

File: cernopendata/modules/fixtures/data/records/cms-tools-Run1-nano-examples.json [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+    },
+    "use_with": {
+      "description": "These examples can be run with the datasets in the nanoaod-run1 or nanoaodsim-run1 formats",
```
[the remark this answers is not in the document]
Comment (jmhogan) [reply cernopendata/modules/fixtures/data/records/cms-tools-Run1-nano-examples.json]: Done

**Review by katilp — APPROVED** [review]
Thanks, looks good.

**Review by tiborsimko — APPROVED** [review]
Thanks, rebased and squashed, added files on EOS and added file information,

**Pull request merged**